### ISSUE TITLE
Make k8s namespace configurable

### ISSUE THREAD

**Issue by JulzDiverse** [julz/cube#24]
We need this to be able to re-use a single k8s cluster for two different pipelines.

**Comment by suhlig** [comment]
Pending merge into julz/cube master

**Comment by suhlig** [comment]
Blocked by #17. We put too many features on top of that feature's branch.
